ovs/lib/alba.py

```diff
@@ -396,14 +396,19 @@ def _add_osds(alba_backend_guid, alba_node_guid, osds, domain, metadata):
             AlbaController._logger.exception('Could not load OSD information.')
             raise
 
+        failure_osds = []
+        unclaimed_osds = []
         # Verify each OSD whether it's already been claimed or just available for claiming
         for osd_list, osd_status in [[claimed_osds, 'claimed'],
                                      [available_osds, 'available']]:
             for actual_osd_info in osd_list:
                 ips = actual_osd_info['ips']
                 port = actual_osd_info['port']
                 decommissioned = actual_osd_info['decommissioned']
-                if port in port_osd_info_map and decommissioned is False:
+                if decommissioned is True:
+                    failure_osds.append('{0}:{1}'.format(ips, port))
```

Comment:
Kinda strange to have a comma-separated list and port. Take first IP in list for string formatting?

Reply:
It's pretty much an ignored value